compiler/src/iree/compiler/Codegen/Common/GPU/test/gpu_lower_to_ukernels.mlir

```diff
@@ -284,3 +293,41 @@ func.func @argmax_ukernel_unsupported_arch(%arg0 : tensor<1x?xf32>) -> tensor<1x
 //      CDNA1-LABEL: func @argmax_ukernel_unsupported_arch(
 //      CDNA1-NOT: iree_codegen.ukernel.generic
 //      CDNA1: linalg.generic
+
+// -----
+
+// Test user-provided bitcode in the source IR.
+
+#gfx942_target = #hal.executable.target<"rocm", "rocm-hsaco-fb", {abi = "hip", iree.gpu.target = #iree_gpu.target<arch = "gfx942", features = "", wgp = <compute = fp32, storage = b32, subgroup = none, dot = none, mma = [], subgroup_size_choices = [], max_workgroup_sizes = [], max_thread_count_per_workgroup = 0, max_workgroup_memory_bytes = 0, max_workgroup_counts = []>>, ukernels = "all"}>
+func.func @argmax_2d_f32i64(%arg0 : tensor<1x?xf32>) -> tensor<1xi64> attributes {
+  hal.executable.target = #gfx942_target,
+  // Dummy bitcode with an unusual length of 12.
+  hal.executable.objects = [#hal.executable.object<{path = "iree_uk_amdgpu_argmax_f32i64.c.gfx942.bc", data = dense<"0x4243C0DE0123456789ABCDEF"> : tensor<12xi8>}>]
```

Comment:
it's best to exactly match the printed output - since it prints as `dense<[66, 67, ...` this should be the same - think about the poor soul who comes along and has to update this test/understand a diff :)